### DISCUSSION TITLE
Background while Symbol-editing

### DISCUSSION THREAD

**Discussion by infernosblade** [keiwando/toonsquid#434] [Q&A]
Is there a way to not have an infinite background while editing a symbol, i cant tell where my canvas ends.

**Comment by Cat-Butler** [comment]
Yes, there is!

1. While editing the symbol bring up the timeline.
2. Select the Background layer (by the way, you can also change the symbol background colour to make symbol editing easier).
3. In the Properties panel ([Inspector](https://toonsquid.com/handbook/layers/inspector/)) for the [Background layer](https://toonsquid.com/handbook/layers/background/), turn off Infinite Background.
